examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -107,10 +107,20 @@ function default_cache(Y, params, spaces, numerics, simulation)
         ghost_buffer =
             (ghost_buffer..., ᶜχρq_tot = Spaces.create_ghost_buffer(Y.c.ρ))
     )
+    if apply_limiter
+        tracers = filter(is_tracer_var, propertynames(Y.c))
+        make_limiter =
+            ᶜ𝕋_name ->
+                Limiters.QuasiMonotoneLimiter(getproperty(Y.c, ᶜ𝕋_name), Y.c.ρ)
+        limiters = NamedTuple{tracers}(map(make_limiter, tracers))
```

Comment:
Just a heads up / looking down the road, this pattern will not currently work for edmf variables because it assumes that every variable can be accessed with a single `getproperty`. You can inspect the nested `propertynames` structure of `Y` in one of the driver examples with `--turbconv edmf`.

We should probably start thinking about how to either

 - Redesign the edmf variables to work with this or
 - Redesign this pattern (of using `getproperty`/`propertynames`) to work with edmf.

It's not clear to me which road we should take, but the incompatibility of these two designs is clear. In addition, I suggest we also consider GPU constraints when deciding, since that could weigh in on the feasibility of these options

Reply:
Thanks for the heads-up. Yes, we should discuss the way forward. This pattern of accessing these variables was co-authored with @dennisYatunin and I was personally not aware of how the EDMF variables are stored, so thank you so much for explaining that and pointing to examples!